src/main/scala/cromwell/binding/types/WdlFileGlobType.scala

```diff
@@ -0,0 +1,33 @@
+package cromwell.binding.types
+
+import cromwell.binding.values.{WdlFile, WdlFileGlob, WdlString}
+import spray.json.JsString
+
+import scala.util.{Success, Try}
+
+// TODO: Do we even need this?
```

Comment:
As far as I can tell it's the `WdlFileGlobType` and `WdlFileGlob` aren't used.  I remember initially resisting but then agreeing at some point that we should have a `WdlFileGlob`, but I don't remember why.  I wonder if this PR missed some functionality in which this would be used.  I think it was something about the file evaluator that JES uses needs to know if it's a glob or just a regular static file path.